mwm-3-kotlin-1/presentation/PITCHME.md

```diff
@@ -1402,6 +1402,222 @@ Java_com_mwm_demo_1kotlin_jni_BridgeJni_getString(
 
 ---
 
+### Scope functions
+
+Note
+
+- There are just some fine details between each scope functions.
+- Functions which help to structure the code by creating blocks of logics.
+- `apply` and `also` return the caller.
+- `run` and `let` return the block result.
+
+---
+
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Run</span> <span style="text-transform: none; font-size:0.8em;">scope function</span>
+
+```kotlin
+private var a = 5
+fun testRun() {
+    run {
+        val a = 10
+        Log.i("ScopeDemo", "a : $a")
+        Log.i("ScopeDemo", "this:a : " + this.a)
+    }
+}
+```
+
+```java
+private int a;
+public final void testRun() {
+    ScopeDemo $receiver = (ScopeDemo)this;
+    int a = 10;
+    Log.i("ScopeDemo", "a : " + a);
+    Log.i("ScopeDemo", "this:a : " + $receiver.a);
+}
+```
+
+Note
```

Comment:
Should be `Note:` otherwise it will not be a speaker note

Reply:
Fixed